official/benchmark/benchmark_uploader_test.py

```diff
@@ -61,7 +61,7 @@ def setUp(self, mock_bigquery):
       json.dump({"model_name": "value"}, f)
 
   def tearDown(self):
-    tf.gfile.DeleteRecursively(self.get_temp_dir())
+    tf.compat.v1.gfile.DeleteRecursively(self.get_temp_dir())
```

Comment:
not 100% sure, but is this the replacement: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/io/gfile/rmtree ?